schema/gene_family.js

```diff
@@ -0,0 +1,27 @@
+import type { GraphQLFieldConfig } from "graphql"
+import { GraphQLObjectType, GraphQLString, GraphQLList } from "graphql"
+import Gene from "./gene"
+
+const GeneFamilyType = new GraphQLObjectType({
+  name: "GeneFamily",
+  fields: {
+    id: {
+      type: GraphQLString,
+    },
+    name: {
+      type: GraphQLString,
+    },
+    genes: {
+      type: new GraphQLList(Gene.type),
+    },
+  },
+})
+
+const GeneFamily: GraphQLFieldConfig<GeneFamilyType, *> = {
+  type: GeneFamilyType,
+  resolve: () => {
+    return root => root
+  },
```

Comment:
Orly?

I once [returned an empty object](https://github.com/artsy/metaphysics/blob/2d9de40d40a537a17447d7094589f8c9065e17ba/schema/home/index.js#L23-L26), but if it’s really not needed at all then can you remove it there too?